lms/static/sass/inline-discussion.scss

```diff
@@ -0,0 +1,33 @@
+// From lms-main-v1.scss
+
+// libs and resets *do not edit*
+@import 'bourbon/bourbon'; // lib - bourbon
+@import 'vendor/bi-app/bi-app-ltr'; // set the layout for left to right languages
+
+// base - utilities
+@import 'base/variables';
+@import 'base/mixins';
+
+// From _build-lms-v1.scss
+
+// app - discussion
+@import "discussion/utilities/variables-v1";
+@import "discussion/mixins";
+@import 'discussion/discussion'; // Process old file after definitions but before everything else, partial is deprecated.
+@import 'discussion/discussion-v1'; // Non-Pattern Library styling
+@import "discussion/elements/actions";
+@import "discussion/elements/editor";
+@import "discussion/elements/labels";
+@import "discussion/elements/navigation";
+@import "discussion/views/home";
+@import "discussion/views/thread";
+@import "discussion/views/create-edit-post";
+@import "discussion/views/response";
+@import 'discussion/utilities/developer';
+@import 'discussion/utilities/shame';
+
+// From _build-course.scss
+
+// discussion
+@import "course/discussion/form-wmd-toolbar";
+@import "course/discussion/form";
```

Comment:
@itsjeyd not 100% sure about this, but is this `course/*` files for inline discussion or for course (aka tab) discussion?